packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -9,8 +9,13 @@ let buildRequire = template(`
 `);
 
 let buildExportsModuleDeclaration = template(`
-  Object.defineProperty(exports, "__esModule", {
-    value: true
+  Object.defineProperties(exports, {
+    __esModule: {
+      value: true
+    },
+    __esExports: {
+      value: $0
```

Comment:
I can change it, but I'll have to find how to properly declare a new variable invisible from the userland code. Is there an easy way to do this?